Clarify `pl-question-panel` text (#8170)
PrairieLearn/PrairieLearn: docs/elements.md

#### Details

> Generally, any text placed in `question.html` will appear in 

Reviewer: Can this description be copied to the other panel elements?

Author: If you were to spend time on this, I think @nwalters512's suggestion is probably most appropriate:

> In a perfect (and hopefully future) world, we'd have documentation that explains panels, and then we can just have the docs for all these elements be something like this:

> > Ensures that content is only displayed in the question panel. See (link to panel documentation) for more details).

Reviewer: Yeahhh, probably not worth fussing over and just waiting for the docs revamp.

Reviewer: > To be clear, my (firm) suggestion was to make the same changes to the other elements _now_, and save the part where we centralize that explanation for some hypothetical future revamp.

That makes sense, I had the same thing in mind when making my original comment. I'll go ahead and copy everything to make things consistent.

Reviewer: @nwalters512 changes made here https://github.com/PrairieLearn/PrairieLearn/pull/8170/commits/1c612d73e6ce5d51522a25a1a5349e4ae3e9dfa6

> Generally, any text placed in `question.html` will appear in
> `question-panel`, `submission-panel`, and `answer-panel`. To prevent
> content from being displayed in the `submission-panel` and
> `answer-panel`, surround that content with the `<pl-question-panel>` tags.

Reviewer 2: Two nits:

- I don't love "text" as it excludes things like PL and HTML elements. Hence, "contents", though bikeshed if you'd like!
- We don't ever refer to `question-panel`, etc. - it's always "the question panel" and so on.

```suggestion
Generally, the contents of `question.html` will appear in
the question panel`, submission panel, and answer panel. To prevent
content from being displayed in the submission panel and
 answer panel, surround that content with the `<pl-question-panel>` tags.
```

#### Details

Information between this tag represents the question text and is 

> always shown when the question is displayed to students. When a 

Reviewer 2: I don't think this is precise enough - we consider the submission and answer panels to also be "displaying the question to students".

An alternative formulation of this text that we can iterate on:

> The contents of this element will only be displayed in the question panel, which generally displays the question prompt and any inputs that will be used to submit answers. The contents will not be shown when the question is rendered to display the correct answer or a particular submission.

Author: Okay, I've taken another crack at it:

> Generally, any text placed in `question.html` will appear in all of `pl-question-panel`, `pl-submission-panel`, and `pl-answer-panel`. To prevent content from being displayed in `pl-submission-panel` and `pl-answer-panel`, surround the content with the `<pl-question-panel>` tags.

Feel free to say you like your attempt better, I can iterate on that instead!

Reviewer 3: This sounds like a confusion between the elements (e.g., pl-submission-panel) with the panels (the submission panel).

Author: Yes you're right, thanks! Made those edits in the comment.

Reviewer 2: Your version works for me! I'll leave a few more comments after you make the change.

I'd recommend ensuring that the docs for the other `*-panel` tags is consistent with this. In a perfect (and hopefully future) world, we'd have documentation that explains panels, and then we can just have the docs for all these elements be something like this:

> Ensures that content is only displayed in the question panel. See (link to panel documentation) for more details).

That can wait for the inevitable docs revamp 🙂

student either makes a submission or receives the correct answer,
the information between these tags is not shown in the 
`submission-panel` and `answer-panel`. If content exists outside of a 

> question panel, then it will be displayed in the `answer-panel`.

Author: also I just realized this isn't true, it should be:

> If content exists outside of a question panel, then it will be displayed in the `answer-panel` **and the `submission-panel`**.
